### DISCUSSION TITLE
[@types/three] XRInputSourceChangeEvent should extend Event

### DISCUSSION THREAD

**Discussion by jdspugh** [DefinitelyTyped/DefinitelyTyped#57005] [Issues with a @types package] [answered]
In `DefinitelyTyped/types/three/src/renderers/webxr/WebXR.d.ts`, `XRInputSourceChangeEvent` should extend `Event` i.e.

`export interface XRInputSourceChangeEvent {`

should become:

`export interface XRInputSourceChangeEvent extends Event {`

[a comment by an automated account is not shown]

**Reply by Methuselah96** [reply]
@jdspugh Can you open an issue on https://github.com/three-types/three-ts-types instead? It's less likely to be forgotten there.

**Comment by jdspugh** [comment]
https://developer.mozilla.org/en-US/docs/Web/API/XRInputSourcesChangeEvent

### Methods
While XRInputSourcesChangeEvent defines no methods of its own, it _inherits_ methods _from_ its parent interface, _Event_.
